README.md

```diff
@@ -1,2 +1,31 @@
 # Personal-Portfolio
+
 This is my Microverse Personal Portfolio.
+
+> Week one Day 3 task
+
+## Built With
+
+- HTML/CSS
+
+## Getting Started
+
+**Setting up My Personal Portolio task**
+
+## Authors
+
+👤 **Promise Uka**
+
+- GitHub: [@githubhandle](https://github.com/Ukaypromise/)
+- Twitter: [@twitterhandle](https://twitter.com/PromiseUkay)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/engr-promise-uka-5a1450173/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] Please add the License section as well from the Microverse ReadMe template, since it is necessary to tell the viewer what permissions they have when using this project. You should additionally add the MIT.md file in your repo as the license.